<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(241, 239, 218);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="song_slider">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QSlider{
border-color: #bcbcbc;
}
QSlider::groove:horizontal { border: 1px solid #999999; height: 1px; margin: 0px 0; left: 5px; right: 5px; }
QSlider::handle:horizontal { border: 0px ;  width: 15px; margin: -7px -7px -7px -7px; }

QSlider::add-page:horizontal{ background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #bcbcbc, stop:0.25 #bcbcbc, stop:0.5 #bcbcbc, stop:1 #bcbcbc); } 

QSlider::sub-page:horizontal{ background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #439cf3, stop:0.25 #439cf3, stop:0.5 #439cf3, stop:1 #439cf3); }</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>20</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
    border-color: #cbcbcb; } QSlider::groove:vertical { background: #cbcbcb; width: 6px; border-radius: 1px; padding-left:-1px; padding-right:-1px; padding-top:-1px; padding-bottom:-1px; } QSlider::sub-page:vertical { background: #cbcbcb; border-radius: 2px; } QSlider::add-page:vertical { background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #439cf4, stop:1 #439cf4); background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1, stop: 0 #439cf4, stop: 1 #439cf4); width: 10px; border-radius: 2px; } QSlider::handle:vertical { border-image: url(:/NiceTalk/Images/Setting/volume_nor.png); margin: -2px -7px -2px -7px; height: 17px; } </string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="MusicList">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>700</width>
      <height>450</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playTime">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>570</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
     </font>
    </property>
    <property name="text">
     <string>0:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="musicTime">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>570</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0:00</string>
    </property>
   </widget>
   <widget class="QListWidget" name="collect">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>781</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
   </widget>
   <widget class="QListWidget" name="like">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>781</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
   </widget>
   <widget class="QListWidget" name="songLibrary">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>200</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
   </widget>
   <widget class="QLabel" name="picLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>85</width>
      <height>85</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0.5px groove #aaaa7f ;border-radius:8px;padding:1px 2px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/image/pic.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 楷体 Std R</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 12pt &quot;Adobe 楷体 Std R&quot;;</string>
    </property>
    <property name="text">
     <string>正在播放：</string>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>535</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 楷体 Std R</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe 楷体 Std R&quot;;</string>
    </property>
    <property name="text">
     <string>还没有播放歌曲哦</string>
    </property>
   </widget>
   <widget class="QWidget" name="LyricsWidget" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>781</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <widget class="QLabel" name="D3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>380</y>
       <width>501</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="U3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>700</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(150, 150, 150);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="U2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>700</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="U1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>700</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="U0">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>200</y>
       <width>700</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="D1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>260</y>
       <width>700</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="D2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>320</y>
       <width>700</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="skin">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>5</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/image/skin.png</normaloff>:/image/skin.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="about">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>5</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/image/about.png</normaloff>:/image/about.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="create">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 楷体 Std R</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Adobe 楷体 Std R&quot;;</string>
    </property>
    <property name="text">
     <string>创建的歌单</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/image/create.png</normaloff>:/image/create.png</iconset>
    </property>
   </widget>
   <widget class="QListWidget" name="create_list">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>160</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="selectionRectVisible">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
